src/active-page/active-page-slice.ts

```diff
@@ -0,0 +1,39 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../app/store';
+import { updateStateFromHistory } from '../url-history/actions';
+import { ActivePage } from './types';
+
+const initialState: ActivePage = 'duplicateSearching' as ActivePage;
```

Comment:
Ah, that's a great idea re: future-proofing, and normally I'd be all for it, but this is actually the one state key I was hoping to keep just a string! The reason why is that it makes it a lot easier to manually update the URL!

Because this effectively controls the site navigation, I could see people setting this query parameter manually a lot. For example, you can navigate to `bugomattic&activePage=reportingFlow` and just jump straight into the reporting flow!

With how we do the state serialization in the URL, if we make it an object, the string unfortunately gets a good bit messier. 😕 

Is that okay with you?

Reply:
> Because this effectively controls the site navigation, I could see people setting this query parameter manually a lot. For example, you can navigate to bugomattic&activePage=reportingFlow and just jump straight into the reporting flow!

That makes sense then! I'm happy with the current implementation. 👍🏻